seneca.js

```diff
@@ -181,7 +181,10 @@ var internals = {
     legacy: {
 
       // use old error codes, until version 3.x
-      error_codes: true
+      error_codes: true,
+
+      // use parambulator for message validation, until version 3.x
+      validate: true
```

Comment:
to clarify this:
until version 3 === false from v3

removal is probably a few versions away - backwards compatibility and all that :)